modules/terraform/Makefile

```diff
@@ -34,6 +34,14 @@ terraform/validate:
 terraform/tflint:
 	tflint --enable-plugin=aws
 
```

Comment:
Please add a comment above this target explaining what it does.

Reply:
done